core/controllers/galleries_test.py

```diff
@@ -125,7 +125,13 @@ def test_gallery_handler_for_created_explorations(self):
             'user_email': self.ADMIN_EMAIL,
             'username': self.ADMIN_USERNAME,
             'search_cursor': None,
-            'profile_picture_data_url': None,
+            'profile_picture_data_url':(
```

Comment:
No need to do this -- if this breaks, I would have no idea how to fix it. Suggest changing self.assertEqual to self.assertDictContainsSubset,